hetmatpy/pipeline.py

```diff
@@ -26,12 +42,51 @@ def add_gamma_hurdle_to_dgp_df(dgp_df):
         )
     # Compute gamma-hurdle parameters
     dgp_df['mean_nz'] = dgp_df['sum'] / dgp_df['nnz']
```

Comment:
Did this have any precision issues where `mean_nz` was nonzero when it should be zero? I guess that would depend on whether there are any issues with `dgp_df['sum']` being nonzero when it should be zero?

Reply:
It doesn't appear to be the case, the smallest mean_nz in the database is .15, and seems pretty evenly distributed up from there.